src/libraries/System.Security.Cryptography.Algorithms/tests/RijndaelTests.cs

```diff
@@ -19,38 +19,38 @@ public class RijndaelTests
         [Fact]
         public static void VerifyDefaults()
         {
+            static void verifyDefaults(Rijndael alg)
```

Comment:
In situations like this, where the test method differs from the local function by the casing of a single letter, it'd be really easy to accidentally get a stack overflow or an infinite loop by calling the wrong thing.  Could you rename such local functions to something a little more different?

Reply:
Sure, I actually started using `test` for the local function at some point, I missed updating some of the ones I did earlier.